### PR TITLE
Exercise_11 Version4

### PR DESCRIPTION
**Pull request by serine-galstyan**

### CODE REVIEW

File: serine.galstyan/repo/Lesson_3/exercise_11/exercise_11.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/evn python3
+
+width = float(input("Enter the width(in feet):"))
+length = float(input("Enter the length(in feet):"))
+field_area = width * length / 43560
```
Comment (Zohrabya) [inline serine.galstyan/repo/Lesson_3/exercise_11/exercise_11.py]: LW: What represents number 43560. Use a constant variable.

**Review by Zohrabya — COMMENTED** [review]
exercise_11: 49 - reply

File: serine.galstyan/repo/Lesson_3/exercise_11/exercise_11.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/evn python3
```
Comment (Zohrabya) [inline serine.galstyan/repo/Lesson_3/exercise_11/exercise_11.py]: CSW: Wrong shebang line

File: serine.galstyan/repo/Lesson_3/exercise_11/exercise_11.py [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+
+width = float(input("Enter the width(in feet): "))
+length = float(input("Enter the length(in feet): "))
+feets_in_an_acre = 43560
```
Comment (Zohrabya) [inline serine.galstyan/repo/Lesson_3/exercise_11/exercise_11.py]: CSW: Constant value

**Review by Zohrabya — COMMENTED** [review]
exercise_11: 47 - reply

File: serine.galstyan/repo/Lesson_3/exercise_11/exercise_11.py (hunk 3 above)
Comment (serine-galstyan) [reply serine.galstyan/repo/Lesson_3/exercise_11/exercise_11.py]: The feets_in_an_acre variable represents 43560 number, as it was indicated to do in the previous comment.

**Review by Zohrabya — APPROVED** [review]
exercise_11: 97 - Accepted

**Pull request merged**